util.ts

```diff
@@ -1,8 +1,12 @@
+import { parseFrontMatterAliases, parseFrontMatterTags } from "obsidian";
+
 let illegalRe = /[\/\?<>\\:\*\|"]/g;
 let controlRe = /[\x00-\x1f\x80-\x9f]/g;
 let reservedRe = /^\.+$/;
 let windowsReservedRe = /^(con|prn|aux|nul|com[0-9]|lpt[0-9])(\..*)?$/i;
 let windowsTrailingRe = /[\. ]+$/;
+let potentialTagsRe = /(#[^ ^#]*)/g; // Finds any non-whitespace sections starting with #
```

Comment:
Let's not mix these into the file name sanitizer code.

Reply:
Actually, can we move all of these util functions into `formats/keep/util.ts`?

Reply:
Done & pushed.